cdap-runtime-spi/src/main/java/io/cdap/cdap/runtime/spi/provisioner/RetryableProvisionException.java

```diff
@@ -29,7 +31,15 @@ public RetryableProvisionException(Throwable cause) {
     super(cause);
   }
 
-  public RetryableProvisionException(String message, Throwable cause) {
-    super(message, cause);
+  public RetryableProvisionException(@Nullable String operationId, Throwable cause) {
+    super(createMessage(operationId), cause);
+  }
+
+  private static String createMessage(@Nullable String operationId) {
+    if (operationId != null) {
+      return String.format("Dataproc operation %s failure", operationId);
```

Comment:
This should be provisioner-agnostic. It's better to have this code within a subsclass and throw that class from Dataproc

Reply:
thanks! fixed.